### ISSUE TITLE
cdf

### ISSUE THREAD

**Issue by vlukas23** [lutraconsulting/qgis-crayfish-plugin#353]

**Comment by PeterPetrik** [comment]
Hi, do you use QGIS 2.18 or QGIS 3.4? Which version of crayfish do you use?

**Comment by vlukas23** [comment]
QGIS 2.18 and crayfish 2.7.2

**Comment by vlukas23** [comment]
are the officially supported crayfish versions only for QGIS 3.4?

**Comment by PeterPetrik** [comment]
Yes, 2.18 life ends in February 2019 and there are many installation problems with crafish for QGIS 2.18 due to such library dependencies.
